sssd_test_framework/hosts/keycloak.py

```diff
@@ -46,6 +46,28 @@ def __init__(self, *args, **kwargs) -> None:
 
         self.adminpw = self.config.get("adminpw", "Secret123")
 
+        self.client.setdefault("id_provider", "idp")
+        self.client.setdefault("enabled", "true")
+        self.client.setdefault("idp_type", "keycloak:https://master.keycloak.test:8443/auth/admin/realms/master/")
+        self.client.setdefault("auto_private_groups", "true")
+        self.client.setdefault("use_fully_qualified_names", "true")
+        self.client.setdefault("debug_level", "9")
```

Comment:
This is definitely not needed. We set the debug level automatically when writing configuration.

Reply:
Removed